cartridge/webui.lua

```diff
@@ -26,9 +26,11 @@ local function get_blacklist()
     return vars.blacklist
 end
 
-local function init(httpd)
-    front.init(httpd)
-    front.add('cluster', front_bundle)
+local function init(httpd, webui_enabled)
+    if webui_enabled then
+        front.init(httpd)
+        front.add('cluster', front_bundle)
+    end
 
     graphql.init(httpd)
```

Comment:
Let's move this single line to `cartridge.lua`. The logic will be the following.

1. If http_enabled, we start httpd and call graphql.init(httpd)
2. If webui_enabled, we call  front.init(httpd) + all graphql.add_callback / add_mutation